sno/cli_util.py

```diff
@@ -53,3 +53,25 @@ def do_json_option(func):
         default=False,
         help="Whether to format the out output as JSON instead the default text output.",
     )(func)
+
+
+def eager_callback_flag(*args, callback, **kwargs):
```

Comment:
There are a couple of other existing eager flags (in sno/cli.py and sno/init.py) , should make them use this also

Reply:
I'd probably take this one step further and make it call `ctx.exit(0)` so the callbacks don't have to. 

And call it something like `call_and_exit`:

```python
@call_and_exit("--version", callback=print_version, help="Print the version number") 
```

Since all our usages of it do use it like that already...

Reply:
Done

Reply:
Done